docs/customization.md

```diff
@@ -424,3 +424,17 @@ def post_build(
         f"{req.name}: running post build hook for {sdist_filename} and {wheel_filename}"
     )
 ```
+
+## Customizations using settings.yaml  
+
+To use predefined urls to download sources from, instead of overriding the entire `download_source` function, a mapping of package to download source url can be provided directly in settings.yaml:
+
+```yaml
+download_source:
+  torch:
+      url: "https://github.com/pytorch/pytorch/releases/download/v${version}/pytorch-v${version}.tar.gz"
+      rename_to: "torch-${version}"
```

Comment:
So this is purposefully only the base filename? What if the source is a zip file (we have some of those)?

Reply:
Right now it is up to the user to set the base name. If it is a zip file then they should name it `torch-${version}.zip`

Reply:
Ah, OK, then the example should be updated to reflect that.

```suggestion
      rename_to: "torch-${version}.tar.gz"
```